resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -696,6 +696,8 @@ sam.init.template.hello_world.description=A basic SAM app
 sam.init.template.hello_world.name=AWS SAM Hello World
 sam.init.template.hello_world_gradle.name=AWS SAM Hello World (Gradle)
 sam.init.template.hello_world_maven.name=AWS SAM Hello World (Maven)
+sam.init.template.hello_world_typescript.description=A sample app using TypeScript
```

Comment:
Should we specify in the description that it's a 'sample Hello World app'?

Reply:
it's more complicated than the standard hello world, which is why the description is more generic